src/System.Private.ServiceModel/tools/test/Bridge/Bridge/ConfigController.cs

```diff
@@ -16,14 +16,25 @@ namespace Bridge
 {
     public class ConfigController : ApiController
     {
-        private const int Default_BridgeIdleTimeoutMinutes = 20;
+        public static EventHandler<Tuple<BridgeConfiguration, BridgeConfiguration>> BridgeConfigurationChanged;
```

Comment:
You don't need to declare your own struct. `EventHandler<TEventArgs>` is just a predefined convenience delegate. It's declared as this:
```csharp
public delegate void EventHandler<TEventArgs>(Object sender, TEventArgs e); 
```
So just define your own delegate type for the events and don't worry about `EventHandler<TEventArgs>`. Something like this:
```csharp
public delegate void ConfigChangeEventHandler(Object sender, BridgeConfiguration previous, BridgeConfiguration updated);
```
As you aren't even using the event handler type and just using anonymous delegates, you could dispense with any custom types altogether and declare the event using `Action<Object, BridgeConfiguration, BridgeConfiguration>`.